crates/sql_provider_datafusion/src/dbconnectionpool/postgrespool.rs

```diff
@@ -42,7 +42,30 @@ impl
         _mode: Mode,
         _params: Arc<Option<HashMap<String, String>>>,
```

Comment:
Rename to `params` now that it's being used. The `_` prefix is only used to indicate that an argument is intentionally unused.

Reply:
Renamed to params in the new commits